core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -238,7 +238,14 @@ export class ResultPanelComponent {
     return columnNames.map(col => ({
       columnDef: col,
       header: col,
-      getCell: (row: IndexableObject) => this.trimTableCell(row[col].toString())
+      getCell: (row: IndexableObject) => {
+        if (row[col]) {
+          return this.trimTableCell(row[col].toString());
+        } else {
+          // allowing null value from backend
+          return 'null';
```

Comment:
I think displaying an empty string instead of null makes more sense. Please change to empty string.

Reply:
changed